pkg/virt-api/rest/console.go

```diff
@@ -107,8 +107,12 @@ func (t *Console) Console(request *restful.Request, response *restful.Response)
 
 	dstAddr := string(pods.Items[0].Status.PodIP)
 
-	// FIXME, don't hardcode virt-handler port. virt-handler should register itself somehow
-	port := "8185"
+	port := string(pods.Items[0].Annotations["virtHandlerPort"])
```

Comment:
What this tries to do, is to keep virt-handler independent of the network namespace. Can be, that we run virt-handler with host network namespace, or pod network namespace. Depending on other design decisions, regarding libvirt, we changed that already back and forth. Depending on which one it uses, we will either have to use the host-port + a free port, which should probably be configurable from the host perspective, vs. running on the Pod, where we can freely pick the port, and use the pod IP. 

I it not 100% certain, but there is a good chance that we will do that switch with virt-handler back to the host network namespace, so that integraing with CNI becomes easier. As part of that libvirt will probably move to it's pod network namespace. @davidvossel what do you think?